docs/project/3-git-remote.md

```diff
@@ -4,94 +4,101 @@
-2. Copy the Git SSH URL of the repository (REPO_URL)
-    <figure markdown>
-    ![Merging](images/repo-url.png){ width="400" height="100" }
-        <figcaption>
-        Copy the Git URL
-        </figcaption>
-    </figure>
+1. Create an empty repository in your GitHub or GitLab account
+2. Copy the SSH URL of the created repository.
 
-!!! note "Note"
-    REPO_URL : https://github.com/mlrepa/project-1-git
-3 - Сlone the repo using the command:
+GitHub:
 
-```bash 
-git clone REPO_URL 
-```
+![Untitled](images/3-git-remote-clone_repo-GitHub.gif)
+
+GitLab:
 
-!!! note "Note"
+![Untitled](images/3-git-remote-clone_repo-GitHub.gif)
+3. Clone the repository using the command `git clone` in the CLI.
 
-    Git automatically creates a folder with the repository name and downloads the files there.
+```bash
+git clone REPO_URL
+```
 
+!!! Note
+        Git automatically creates a folder with the repository name and downloads the files there.
 
+### **2. Create and Update `dev` branch, push it to the remote**
 
-### **2. Update and push to remote**
+Now let’s see how to work with a remote repository using GitHub. 
 
-Let’s see how to work with a remote repository using GitLab.
+1. First, we navigate to our **3-git-remote** directory using the command `cd 3-git-remote`.
 
-1. First, we navigate to our project-1-git directory using the command `cd project-1-git`.
-   
-```bash 
+```bash
 # Go to the repository 
-cd project-1-git 
+cd 3-git-remote
 ```
-2. Then, we create a new branch called `dev`. By this way you can make changes without affecting the main codebase. And we switch to new branch with the command `git checkout -b dev`.
-   
-```bash 
-# Create `dev` branch and checkout to it
+2. Then, we need to create a new branch called `dev`. In this way, you can make changes without affecting the main codebase.
+And we immediately switch to a new branch with the command `git checkout -b dev`.
+
+```bash
+# Create `dev` branch and checkout to it (-b flag)
 git checkout -b dev
 ```
-3. Next, we create a new file called `config.yaml` using the `touch` command and add a line to it using `echo "repo: project-1-git" >> config.yaml`. We confirm the changes using `cat config.yaml`.
- 
-```bash 
-#Create a config file 
+3. Next, we create a new file called `config.yaml` using the `touch` command and add a line to it using `echo "repo: project-1-git" >> config.yaml`. 
+We confirm the changes using `cat config.yaml`.  This command will print the file’s content.
+
+```bash
+# Create a config file, make changes, confirm them
 touch config.yaml 
 echo "repo: project-1-git" >> config.yaml
 cat config.yaml
 ```
-4. Also, we add and commit the changes to the local repository using `git add config.yaml && git commit -m "Add config.yaml"`.
-   
-```bash 
-# Commit changes
+4. Also, we add and commit the changes to the local repository using
+ `git add config.yaml && git commit -m "Add config.yaml"`.
```

Comment:
This command duplicates a command in the code block below. 
Should we remove it?